src/blocks/calendar.scss

```diff
@@ -0,0 +1,54 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: calc(
+    #{$width-cell * $cells-in-row} + #{$gap-size * ($cells-in-row - 1)}
```

Comment:
In the width calculation, you should use parentheses to ensure the correct order of operations when mixing multiplication and addition. [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)